resource/basemodui_latam.txt

```diff
@@ -2222,501 +2222,501 @@
+"L4D_vertical_sync_enabled3_tip"		"Sin artefactos visuales ni problemas de rendimiento. Requiere mucha memoria de la tarjeta gráfica."
+"[english]L4D_shader_detail_tip"		"Adjust graphic levels based on graphics card performance."
+"L4D_shader_detail_tip"		"Ajustar los niveles de gráficos en función del rendimiento de la tarjeta gráfica."
+"[english]L4D_effect_detail_tip"		"Adjust effect levels based on CPU performance."
+"L4D_effect_detail_tip"		"Ajustar los niveles de efectos en función del rendimiento de la CPU."
+"[english]L4D_model_texture_tip"		"Adjust the detail of models and textures in the game."
+"L4D_model_texture_tip"		"Ajustar el detalle de los modelos y las texturas del juego."
+"[english]L4D_paged_pool_mem_tip"		"Configure your driver to allocate less or more memory."
+"L4D_paged_pool_mem_tip"		"Configurar tu unidad para asignar más o menos memoria."
+
+// Audio Options
+"[english]L4D_spkr_config_tip"		"Configure speaker layout and channels."
+"L4D_spkr_config_tip"		"Ajustar la configuración de los altavoces y los canales."
+"[english]L4D_sound_qual_tip"		"Adjust sound quality."
+"L4D_sound_qual_tip"		"Ajustar la calidad de sonido."
+"[english]L4D_enable_voice_tip"		"Allow communication between you and other players."
+"L4D_enable_voice_tip"		"Permitir la comunicación con otros jugadores."
+"[english]L4D_disabled_voice_tip"		"Enable voice in this game to adjust this setting."
+"L4D_disabled_voice_tip"		"Activar la voz en este juego para ajustar esta configuración."
+"[english]L4D_disabled_threshold_tip"		"Use open microphone voice communication style to adjust this setting."
+"L4D_disabled_threshold_tip"		"Utiliza el estilo de comunicación por voz de micrófono abierto para ajustar esta configuración."
+"[english]L4D_voice_comm_tip"		"Select a preference for how you communicate in-game."
+"L4D_voice_comm_tip"		"Seleccionar cómo prefieres comunicarte durante la partida."
+"[english]L4D_voice_transmit_tip"		"Adjust the volume of outgoing communication."
+"L4D_voice_transmit_tip"		"Ajustar el volumen de las comunicaciones salientes."
+"[english]L4D_disabled_voice_transmit_tip"		"Voice transmit volume setting is currently unavailable."
+"L4D_disabled_voice_transmit_tip"		"En este momento no se puede ajustar el volumen de transmisión de voz."
+"[english]L4D_voice_receive_tip"		"Adjust the volume of incoming communication."
+"L4D_voice_receive_tip"		"Ajustar el volumen de las comunicaciones entrantes."
+"[english]L4D_boost_mic_gain_tip"		"Enable or disable a boost to your microphone signal."
+"L4D_boost_mic_gain_tip"		"Activar o desactivar un potenciador de la señal del micrófono."
+"[english]L4D_mic_sens_tip"		"Microphone sensitivity when the open mic setting is enabled."
+"L4D_mic_sens_tip"		"Sensibilidad del micrófono cuando la configuración de micrófono abierto está activada."
+"[english]L4D_test_mic_tip"		"Test whether your microphone is functioning properly."
+"L4D_test_mic_tip"		""Probar si tu micrófono funciona correctamente."
```

Comment:
```suggestion
"L4D_test_mic_tip"		"Probar si tu micrófono funciona correctamente."
```

line 2418 of basemodui contain an extra quote mark. perhaps it is the reason of build error

Reply:
> line 2418 of basemodui contain an extra quote mark. perhaps it is the reason of build error

Thanks man. But i think it's because this language not enabled in translation sync tool so it might be fixed later after Ben add it to the list.

Reply:
alright

Reply:
I just wanted to get rid of the build error. Now if by any mean the amount of quote mark is in even number I may not be able to locate it myself, as I used inspect element to locate it instead of some special way. But here are 3 more quote mark error that I found, which I will send it here because I don't want to create another review

in "chat", filter_joinleave contain an extra quote mark
in "gameui", i found that GameUI_MouseFilter also contain an extra one
in "valve", there is also issue at Valve_HDRDEMO_LeftTitle, but instead it's a missing quote mark

I might not be able to help any further if you apply these fixes and it still show error